source/portable/NetworkInterface/DriverSAM/NetworkInterface.c

```diff
@@ -1126,14 +1125,12 @@ void vNetworkInterfaceAllocateRAMToBuffers( NetworkBufferDescriptor_t pxNetworkB
 static void prvEMACHandlerTask( void * pvParameters )
 {
     UBaseType_t uxCount;
-    UBaseType_t uxLowestSemCount = GMAC_TX_BUFFERS + 1;
```

Comment:
This variable is unused, but it shadows a variable declared on line 1065. If this was intended to modify the global variable, it is not doing so. If it was not intended to modify the global variable, I'd definitely remove it.

Reply:
I believe it'll be better to update this line to modify the global variable - `uxLowestSemCount` for the diagnostic info checks in the `vCheckBuffersAndQueue` to work correctly.

Reply:
Sounds good. Now the line updates the global.